dev-python/dukpy/dukpy-0.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="JavaScript in Python"
+DESCRIPTION="dukpy is a JavaScript runtime environment for Python (tested with python 2.7 and >= 3.4) using the duktape embeddable JavaScript engine. With dukpy, you can run JavaScript in Python."
```

Comment:
Keep description in the limit of 80 characters, check the guidelines

Reply:
ok fixed